plugins/exodus/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ include(GNUInstallDirs)
 
 # Check if the plugin is built externally
 if(PROJECT_IS_TOP_LEVEL)
-  find_package(f3d REQUIRED)
+  find_package(f3d REQUIRED plugin)
```

Comment:
```suggestion
  find_package(f3d REQUIRED COMPONENTS plugin)
```

Reply:
Hmmm you're right, it can be omitted. So the question is if it's more clear with the `COMPONENTS` keyword. I'd say yes personally but I'm fine if you decide to omit it.